DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -54,6 +54,7 @@ FROM Tutorials
 
                          LEFT JOIN Tutorials AS NextTutorials
                          ON NextTutorials.SectionID = Tutorials.SectionID
+                            AND NextTutorials.ArchivedDate IS NULL

```

Comment:
No need to check archived date of the tutorial in the join below? Or of the section in the two joins on the sections table?

Reply:
Yes we need to check the archived dates for that `NextSectionTutorial`. The first section join is the current section, so is checked with the change in the previous markup. The next section is now being checked, thank you for that! All done in #154 🙂